packages/client/src/pages/SignIn/index.tsx

```diff
@@ -1,3 +1,40 @@
+import { Link } from 'react-router-dom';
+import { RouterList } from '../../router/routerList';
+import { useAuth } from '../../hooks/usuAuth';
+import type { RequestSignIn } from '../../store/auth/type';
+
+const mockParams: RequestSignIn = {
+  login: 'Xxxxx',
+  password: '123XXXxxx',
+};
+
 export default function SignIn() {
-  return <>SignIn</>;
+  const { logout, signIn, checkIsAuth } = useAuth();
+
+  const handleCheckIsAuth = () => {
+    checkIsAuth();
+  };
+
+  const handleSignIn = () => {
+    signIn(mockParams);
+  };
+
+  const handleLogout = () => {
+    logout();
+  };
+
+  return (
+    <>
+      SignIn
+      <button onClick={handleCheckIsAuth}>Проверить авторизацию</button>
+      <button onClick={handleSignIn}>Авторизоваться</button>
+      <button onClick={handleLogout}>Выход</button>
```

Comment:
```suggestion
      <button onClick={logout}>Выход</button>
```
Здесь тоже не нужна доп. функция